ios/Podfile

```diff
@@ -87,5 +87,16 @@ abstract_target 'MindloggerMobileCommonPods' do
       config[:reactNativePath],
       :mac_catalyst_enabled => false
     )
+
+    if ENV['LOCAL_DEV_PATCHES'] == 'true'
+      puts '[Local Dev] Removing .xcode.env.local'
+      %x(rm -f #{Pod::Config.instance.installation_root}/.xcode.env.local)
+
+      puts '[Local Dev] Patching YogaKit to fix YGMeasureView call signature'
+      %x(patch #{Pod::Config.instance.installation_root}/Pods/YogaKit/YogaKit/Source/YGLayout.m -N < #{Pod::Config.instance.installation_root}/patches/YGLayout.patch)
+
+      puts '[Local Dev] Patching YogaKit to fix FlipperTransportTypes'
```

Comment:
Granted this was my mistake, this should read “Patching Flipper” instead of YogaKit